app/controllers/application_controller.rb

```diff
@@ -41,7 +41,7 @@ def not_found(_error = nil)
   private
 
   def cookies_preference_set?
-    cookies["consented-to-cookies"].present?
+    cookies["consented-to-analytics-cookies"].present? && cookies["consented-to-marketing-cookies"].present?
```

Comment:
Users see the cookies banner unless analytics and marketing cookies consent is answered.
This will make existing users who `consented-to-cookies` before this PR answer again for their cookies settings.